week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,18 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  var result = price * 1.4;
+  var convertFee = result * 0.01;
+  result += convertFee;
+  return result.toFixed(2);
+}
+function convertToBRL(price) {
+  var result = price * 4.93;
+  var convertFee = result * 0.01;
```

Comment:
Can you think of a way to apply the DRY principle (don't repeat yourself) here? This is a very good example of code that can be improved through DRY – you have 2 identical lines of code!